datadog_checks_dev/datadog_checks/dev/tooling/commands/env/ls.py

```diff
@@ -6,19 +6,26 @@
 from ...e2e import get_configured_checks, get_configured_envs
 from ...testing import get_available_tox_envs
 from ...utils import get_testable_checks
-from ..console import CONTEXT_SETTINGS, echo_info, echo_success
+from ..console import CONTEXT_SETTINGS, echo_info, echo_success, echo_warning
 
 
 @click.command(context_settings=CONTEXT_SETTINGS, short_help='List active or available environments')
 @click.argument('checks', nargs=-1)
 def ls(checks):
     """List active or available environments."""
     if checks:
-        checks = sorted(get_testable_checks() & set(checks))
+        testable_checks = get_testable_checks()
 
         for check in checks:
-            envs = get_available_tox_envs(check, e2e_only=True)
+            if check not in testable_checks:
+                echo_warning(
+                    '{}: not a testable check. Skipped. Testable checks are: {}'.format(
```

Comment:
isnt the list of testeable checks too long?

Reply:
This only happen when people submit an invalid check, so it should be fine IMO.

Reply:
I think it is unnecessary you're basically saying "your check is not ok, all the others are"

Reply:
Yes please don't do this

Reply:
![Capture](https://user-images.githubusercontent.com/9677399/64121309-708d9600-cd6c-11e9-8875-a31c5add0d56.PNG)

Reply:
I'm doubting myself too about the necessity of this change now. Closing then :)